app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    file_names = command.split(" ")
+
+    if file_names[0] != "mv" or len(file_names) != 3:
+        return
+    with open(file_names[1], "r") as file:
+        text = file.read()
+    os.remove(file_names[1])
+
+    file_path = file_names[2].split("/")
```

Comment:
see checklist and refactor your code accordingly

Use os.path.join to concatenate parts of the path. In that case, your app will be cross-platform and will use either / or \ depending on OS